### PR TITLE
Update OWNERS files

### PR DESCRIPTION
**Pull request by valaparthvi**
**What type of PR is this:**
/kind tests

**What does this PR do / why we need it:**
This PR updates the OWNERS list, updating it with the current team members.

**Which issue(s) this PR fixes:**

Fixes #

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

File: openshift-ci/build-root/OWNERS [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
 
 approvers:
-- mohammedzee1000
+- anandrkskd
+- feloy
 
 reviewers:
-- mohammedzee1000
+- anandrkskd
+- feloy
```
Comment (valaparthvi) [inline openshift-ci/build-root/OWNERS:9]: @anandrkskd @feloy
I am unsure about this one, but you both seem to have done some work on this front, so I added you both as owners. Let me know if I should change it.

File: tests/OWNERS [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 - anandrkskd
 - rnapoles-rh
 
 reviewers:
 - rnapoles-rh
 - anandrkskd
-- mohammedzee1000
 - valaparthvi
 - feloy
 - mik-dass
```
Comment (dharmit) [inline tests/OWNERS:12]: Can you remove Mrinal's name from the list?

Comment (valaparthvi) [inline tests/OWNERS:12]:
```suggestion
- valaparthvi
- feloy
```

**Pull request merged**